### PR TITLE
🌱 test/e2e: increase Ginkgo poll progress after to 1h

### PR DESCRIPTION
**Pull request by sbueringer**
Signed-off-by: Stefan Büringer buringerst@vmware.com

**What this PR does / why we need it**:
Currently we get a progress report after 10m every 1m. Unfortunately it's not that useful as it just prints the current stack (e.g. https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api/7871/pull-cluster-api-e2e-full-release-1-3/1612472872549224448/build-log.txt)

Let's increase the timeout so we only get the report if something is stuck for >= 1h.

xref: https://kubernetes.slack.com/archives/C8TSNPY4T/p1673336168049309

Cherry-picking to other supported branches to make e2e test debugging easier there as well.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -82,7 +82,7 @@ GINKGO_FOCUS ?=
 GINKGO_SKIP ?=
 GINKGO_NODES ?= 1
 GINKGO_TIMEOUT ?= 2h
-GINKGO_POLL_PROGRESS_AFTER ?= 10m
+GINKGO_POLL_PROGRESS_AFTER ?= 60m
```
Comment (fabriziopandini) [inline Makefile:85]: What about bumping also the progress interval to 5 minutes?
In the end, if I got this right, we are using this feature only to check that the jobs are not stuck, so we don't really need a stack trace every minute

Comment (sbueringer) [reply Makefile:85]: Sounds good

Comment (sbueringer) [reply Makefile:85]: Done

**Comment by sbueringer** [comment]
Doesn't exist on release-1.2 so we don't need a cherry-pick for that branch.

**Pull request merged**